packages/slice-machine/src/hooks/useSMTracker.ts

```diff
@@ -71,5 +71,9 @@ async function trackPageView(): ReturnType<typeof telemetry.track> {
     title: document.title,
     referrer: document.referrer,
     adapter,
+    outerWidth: window.outerWidth,
```

Comment:
So for me:

- `window.screen.width`: global screen size of your real screen, will not change on browser resize
- `window.outerWidth`: window size of the browser, will change on browser resize

But the most important would be:

- `window.innerWidth` no? It would give you the actual browser window size of the opened page, if you open dev tools it will give you the space specific for SM. Why not using it?